From randy.dunlap@oracle.com  Tue Mar 10 21:09:23 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Tue, 10 Mar 2009 11:59:36 -0700
Subject: Staging: p9auth: fix dependency/build error
To: Stephen Rothwell <sfr@canb.auug.org.au>
Message-ID: <49B6B898.6010902@oracle.com>

From: Randy Dunlap <randy.dunlap@oracle.com>

Fix p9auth dependency/build failure.  It needs to depend on
CRYPTO.

p9auth.c:(.text+0x107297): undefined reference to `crypto_alloc_base'
p9auth.c:(.text+0x1073d4): undefined reference to `crypto_destroy_tfm'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Ashwin Ganti <ashwin.ganti@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/p9auth/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/p9auth/Kconfig
+++ b/drivers/staging/p9auth/Kconfig
@@ -1,6 +1,7 @@
 config PLAN9AUTH
 	tristate "Plan 9 style capability device implementation"
 	default n
+	depends on CRYPTO
 	help
 	  This module implements the Plan 9 style capability device.
 
